docs/readmes/bazel/agw_with_bazel.md

```diff
@@ -0,0 +1,542 @@
+
+#### Docker Bazel cache
+
+The Docker Bazel caches are Docker containers that contain pre-built Bazel caches for all relevant [targets](#targets) and [configurations](#configurations). The containers are created in the [workflow](#workflows) `docker-builder-devcontainer.yml` on a regular basis. The Docker Bazel cache containers are based on the [Bazel-base](#environments) Docker container with the addition of the Bazel caches. The images are published at:
+
+- `ghcr.io/magma/magma/bazel-cache-plain:latest`
+- `ghcr.io/magma/magma/bazel-cache-asan:latest`
+- `ghcr.io/magma/magma/bazel-cache-prod:latest`
+
+The Docker Bazel caches were introduced in [#14562](https://github.com/magma/magma/issues/14562) - this issue also contains a detailed slide with information on the Docker Bazel cache setup.
+
+#### Bazel remote cache
+
+> Info: As of the 4th of January 2023 the Bazel remote cache for Magma is **deprecated**. The tear-down is documented in the issue [#14796](https://github.com/magma/magma/issues/14796). In CI only [Docker images containing pre-built Bazel caches](#docker-bazel-cache) are used.
+
+Information on the Bazel remote caching infrastructure for Magma can be found in the [magma/ci-infra/bazel/remote_caching](https://github.com/magma/ci-infra/blob/master/bazel/remote_caching/Readme.md) repository (CI codeowner access only). This information in particular details how the remote cache can be deployed on AWS using Terraform. General information on Bazel remote caching can be found in the [official Bazel documentation](https://bazel.build/remote/caching).
+
+Furthermore there are GitHub Wiki pages with information on [how to use a deployment of the remote cache](https://github.com/magma/magma/wiki/Bazel-remote-caching) and on [how to recover from a remote cache failure](https://github.com/magma/magma/wiki/Bazel-remote-caching-disaster-recovery-plan).
+
+### Commands
+
+The Bazel commands are structured around verbs like [`build`](#building), [`test`](#testing) or [`query`](#querying). The complete list of Bazel commands can be found in the [official command reference](https://bazel.build/run/build#available-commands). The following sections go into more detail on the most common commands.
+
+## Building
+
+To build something with Bazel, use the command [`bazel build`](https://bazel.build/run/build) and then specify the [targets](#targets) that you want to build, e.g. `bazel build //lte/gateway/python/magma/mobilityd:mobilityd`. When specifying targets on the command line the leading `//` are optional, but they make it clearer that a target is referenced. The prefix `//` is required in build files when a dependency is referenced.
+
+Building happens in a sandbox, i.e. various host properties are not present at build time. This includes environment variables and stdin. The build artifacts can be found in `$MAGMA_ROOT/bazel-bin`, e.g. `$MAGMA_ROOT/bazel-bin/lte/gateway/python/magma/mobilityd/mobilityd`.
```

Comment:
I would have benefited from a sentence more on targets. And IMO could the "//" handling moved to the CLI section.

Reply:
Moved the "//" handling moved to the CLI section.

What more would you have hoped to find in this first part of the `Building` section? Targets are covered in some detail right below in the `Targets` subsection.